browser/src/layer/tile/CanvasSectionContainer.ts

```diff
@@ -1667,6 +1670,9 @@ class CanvasSectionContainer {
 		this.canvas.style.width = cssWidth.toFixed(4) + 'px';
 		this.canvas.style.height = cssHeight.toFixed(4) + 'px';
 
+		app.canvasSize.pX = newWidth;
```

Comment:
above we have some calculations we could already avoid by using SimplePoint created from newWidth / newHeight probably

Reply:
Those lines are for canvas resizing and carefully set for crisp images. Our simplePoint and simpleRectangle classes round the values. Canvas sizing uses floating numbers. I think they are good for now, unless we are really sure that it wouldn't break image quality. I also commented about this in geometry.ts file. Thanks for the comment.